lib/castle_web/views/api/episode_view.ex

```diff
@@ -16,17 +16,16 @@ defmodule CastleWeb.API.EpisodeView do
     }
   end
 
-  def render("show.json", %{conn: conn, episode: episode, trends: trends, meta: meta}) do
-    episode_json(episode, trends, conn)
-    |> put_podcast_link(conn, episode)
```

Comment:
Unfortunately, since i converted this from a big json blob in redis to hash lookup ... we no longer know what podcast an episode belongs to.  But I'm pretty sure we weren't using this functionality yet anyways.